ngi_reports/utils/entities.py

```diff
@@ -419,200 +627,293 @@ def populate(self, log, organism_names, **kwargs):
+                        "qval": qval,
+                        "reads": pfrd,
+                        "bases": base,
+                    }
 
                 except (TypeError, ValueError, AttributeError) as e:
-                    log.warn('Something went wrong while fetching Q30 for sample {} with '
-                             'barcode {} in FC {} at lane {}. Error was: \n{}'.format(sample, barcode, fcObj.name, lane, e)
-                             )
+                    log.warn(
+                        "Something went wrong while fetching Q30 for sample {} with "
+                        "barcode {} in FC {} at lane {}. Error was: \n{}".format(
+                            sample, barcode, fcObj.name, lane, e
+                        )
+                    )
                     pass
                 # Collect lanes of interest to proceed later
-                fc_lane_summary_lims = fc_details.get('lims_data', {}).get('run_summary', {})
-                fc_lane_summary_demux = fc_details.get('illumina', {}).get('Demultiplex_Stats', {}).get('Lanes_stats', {})
+                fc_lane_summary_lims = fc_details.get("lims_data", {}).get(
+                    "run_summary", {}
+                )
+                fc_lane_summary_demux = (
+                    fc_details.get("illumina", {})
+                    .get("Demultiplex_Stats", {})
+                    .get("Lanes_stats", {})
+                )
                 if lane not in fcObj.lanes:
                     laneObj = Lane()
-                    lane_sum_lims = fc_lane_summary_lims.get(lane, fc_lane_summary_lims.get('A', {}))
-                    lane_sum_demux = [d for d in fc_lane_summary_demux if d['Lane'] == str(lane)][0]
+                    lane_sum_lims = fc_lane_summary_lims.get(
+                        lane, fc_lane_summary_lims.get("A", {})
+                    )
+                    lane_sum_demux = [
+                        d for d in fc_lane_summary_demux if d["Lane"] == str(lane)
+                    ][0]
                     laneObj.id = lane
-                    laneObj.cluster = '{:.2f}'.format(round(float(lane_sum_demux.get('PF Clusters', '0').replace(",", ""))/1000000, 2))
-                    laneObj.avg_qval = '{:.2f}'.format(round(float(lane_sum_demux.get('% >= Q30bases', '0.00')), 2))
-                    laneObj.set_lane_info('fc_phix', '% Error Rate', lane_sum_lims, str(r_num))
-                    if kwargs.get('fc_phix',{}).get(fcObj.name, {}):
-                        laneObj.phix = kwargs.get('fc_phix').get(fcObj.name).get(lane)
-                    #Calculate weighted Q30 value and add it to lane data
+                    laneObj.cluster = "{:.2f}".format(
+                        round(
+                            float(
+                                lane_sum_demux.get("PF Clusters", "0").replace(",", "")
+                            )
+                            / 1000000,
+                            2,
+                        )
```

Comment:
Cpuld we set line length to a little bit more than it is currently?

Reply:
I split it up instead